src/Layout/toolset-tasks/OverrideAndCreateBundledNETCoreAppPackageVersion.cs

```diff
@@ -123,6 +123,12 @@ void CheckAndReplaceAttribute(XAttribute attribute)
                 crossgen2Rids.Value += ";osx-x64";
             }
 
+            // Currently has too many RIDs. We only want to include the RIDs of actual packages we
+            // produce, as if don't try to download a missing package then restore will succeed and
+            // we can give a better error during publish that the given RID is not supported.
+            var ilcompilerRids = itemGroup.Elements(ns + "KnownILCompilerPack").First().Attribute("ILCompilerRuntimeIdentifiers");
+            ilcompilerRids.Value = "linux-x64;linux-musl-x64;linux-arm64;linux-musl-arm64;win-x64;win-arm64;osx-x64";
```

Comment:
Ideally we shouldn't hard-code the supported RuntimeIdentifiers in a C# task.  What's the reason for the mismatch between the values coming from the item and what's actually supported?

Reply:
The mismatch is that we were trying to avoid blocking people who wanted to use unsupported, but available packages.

But being too broad here actually causes problems because attempting to download a missing package blocks restore/build.

Reply:
We may want to have `browser-wasm` here for NativeAOT-LLVM support.